src/Common/likePatternToRegexp.cpp

```diff
@@ -16,7 +16,7 @@ String likePatternToRegexp(std::string_view pattern)
     res.reserve(pattern.size() * 2);
 
     const char * pos = pattern.data();
-    const char * const end = pattern.begin() + pattern.size();
+    const char * const end = pattern.data() + pattern.size();
```

Comment:
Funny how this worked before. Couldn't find a hint in the libcxx19 release notes that they disabled the implicit cast from iterators to pointers.

Reply:
Yes, exactly. I'm not even sure I'm doing the right thing here.

Reply:
No worries, replacing `.data` by `.begin` is correct..